app/models/order.rb

```diff
@@ -0,0 +1,9 @@
+class Order < ApplicationRecord
+  belongs_to :customer
+  validates_associated :customer
+
+  validates :product_name, presence: true
+  validates :product_count, presence: true
```

Comment:
We can validate the presence of product_name and product_count in one line with `validates_presence_of :face`